app/lib/zeebe-api.js

```diff
@@ -168,6 +168,7 @@ class ZeebeAPI {
 
     const {
       endpoint,
+      variables,
```

Comment:
Let's add to the [JSDoc](https://github.com/camunda/camunda-modeler/pull/2706/commits/b2d68914c514fb7dac891541bda0b7de0c630fd8#diff-6a98303f67d896274406116f4ffa96443414d8a631d68263fca6678d16c31ebfL164)

Reply:
What is the JSDoc?

Reply:
https://github.com/camunda/camunda-modeler/pull/2706/files/b2d68914c514fb7dac891541bda0b7de0c630fd8#diff-6a98303f67d896274406116f4ffa96443414d8a631d68263fca6678d16c31ebfR164

Reply:
the comment in Line 164 of that same file :-)

Reply:
I also added the `endpoint` which was missing